board/batocera/fsoverlay/etc/init.d/S18governor

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if ! { [ -e /sys/devices/system/cpu/cpufreq/policy0/scaling_governor ] && [ -e /sys/devices/system/cpu/cpufreq/policy0/scaling_available_governors ]; }
+then
+  exit 0
+fi
+
+case "$1" in
+  start|restart|reload)
+    if ! echo "$(/usr/bin/batocera-settings-get system.cpu.governor)" > /sys/devices/system/cpu/cpufreq/policy0/scaling_governor 2>/dev/null
```

Comment:
cpu_governor=$(/usr/bin/batocera-settings-get system.cpu.governor)
if [ $? -eq 0 ] ... or [ -n "$cpu_governor" ] or... ; then
   echo $cpu_governor > .....
else
   /usr/bin/batocera-settings-set system.cpu.governor "$(cat /sys/devices/system/cpu/cpufreq/policy0/scaling_governor)"
fi

Reply:
This is for better readability, maybe only for me

But this would catch all cases, and with the filecheck you inserted it's useable for all platforms that do not make use of this. But the question remains if it is wanted for all platforms then